build fail looks related
```
======================================================================
ERROR: test_compare_as_string_option (corehq.apps.userreports.tests.test_report_filters.DateFilterTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/userreports/tests/test_report_filters.py", line 106, in test_compare_as_string_option
    self.assertEqual(type(val['startdate']), datetime)
KeyError: 'startdate'
----------------------------------------------------------------------
```

--- a/corehq/apps/userreports/tests/test_report_filters.py
+++ b/corehq/apps/userreports/tests/test_report_filters.py
@@ -103,10 +103,10 @@ class DateFilterTestCase(SimpleTestCase):
             return filter.create_filter_value(reports_core_value).to_sql_values()
 
         val = get_query_value(compare_as_string=False)
-        self.assertEqual(type(val['startdate']), datetime)
+        self.assertEqual(type(val['my_slug_startdate']), datetime)
 
         val = get_query_value(compare_as_string=True)
-        self.assertEqual(type(val['startdate']), str)
+        self.assertEqual(type(val['my_slug_startdate']), str)
 
 
 class NumericFilterTestCase(SimpleTestCase):

